data_validation/combiner.py

```diff
@@ -25,17 +25,28 @@
 import ibis
 import ibis.expr.datatypes as dt
 
+from typing import TYPE_CHECKING
 from data_validation import consts
+from data_validation.consts import (
```

Comment:
We've already import consts on the line above. I think it is better to use consts.VALIDATION_STATUS and not import VALIDATION_STATUS directly.